lore/settings.py

```diff
@@ -224,6 +224,7 @@ def get_var(name, default):
 EMAIL_HOST_USER = get_var('LORE_EMAIL_USER', '')
 EMAIL_HOST_PASSWORD = get_var('LORE_EMAIL_PASSWORD', '')
 EMAIL_USE_TLS = get_var('LORE_EMAIL_TLS', False)
+EMAIL_SUPPORT = get_var('LORE_SUPPORT_EMAIL', 'mitx-support@mit.edu')
```

Comment:
Not sure if we want to default this to a real address, maybe `support@example.com` as the default